doc/releases/reef.rst

```diff
@@ -5,6 +5,24 @@ Reef
 Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
+
+v18.2.2 Reef
+============
+
+This is a hotfix release that resolves several flaws and has fixes for Prometheus crash
+and Fix encoder
+
+Notable Changes
+---------------
+
+* mgr/Prometheus: refine the orchestrator availability check to prevent against crashes in the Prometheus module during startup. Introduce additional checks to handle daemon_ids generated within the Rook environment, thus preventing potential issues during RGW metrics metadata generation.
+
+Changelog
+---------
+
+* backport mgr/prometheus: fix orch check to prevent Prometheus crash (`pr#55491 <https://github.com/ceph/ceph/pull/55491>`_, Redouane Kachach)
```

Comment:
"backport" is not a component ;)
```suggestion
* mgr/prometheus: fix orch check to prevent Prometheus crash (`pr#55491 <https://github.com/ceph/ceph/pull/55491>`_, Redouane Kachach)
```

Reply:
@rkachach can you amend the commit comment in the PR pr @idryomov ^^^ and I will fix here?

Reply:
The PR is merged, so the commit title can't be updated.  Let's just fix it on this line here.